.github/workflows/ci.yaml

```diff
@@ -322,6 +322,51 @@ jobs:
           set -euxo pipefail
           mv linux/rust.require.cmake linux/rust.cmake
           flutter test -d linux integration_test/main.dart --verbose
+  
+  dart_web_test:
+    name: Dart (Web) unit tests
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dart-lang/setup-dart@v1
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          target: wasm32-unknown-unknown
+          components: rust-src,rustfmt
+      - name: Install ffigen
+        run: dart pub global activate ffigen
+      - name: Cache dependencies
```

Comment:
Good idea - Shall we add an issue about adding cache for all other CI steps?

Reply:
I don't know; web was particularly annoying because of having to install wasm-pack and downloading Chromium on every test, which was my main motivation for caching. Do any other tests suffer from the same problem?

Reply:
At least suffer from downloading a whole Dart/Rust/llvm/dependency/... Anyway that is not a major problem :)